multimedia/scrcpy/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        Genymobile scrcpy 1.17 v
+github.tarball_from releases
+revision            0
+
+categories          multimedia
+platforms           darwin
+license             Apache-2
+maintainers         {bochtler.io:macports @MarcelBochtler} \
+                    openmaintainer
+description         Display and control your Android device
+long_description    This application provides display and control of Android \
+                    devices connected on USB (or over TCP/IP). It does not \
+                    require any root access.
+
+distfiles-append scrcpy-server-v${version}
```

Comment:
Minor nitpick, but for this line, can we match the alignment of the other lines in this file?

Reply:
Done